resources/lang/id/model_validation.php

```diff
@@ -0,0 +1,115 @@
+    'wrong_confirmation' => 'Konfirmasi tidak cocok.',
+
+    'beatmap_discussion_post' => [
+        'discussion_locked' => 'Diskusi terkunci',
+        'first_post' => 'Tidak dapat menghapus postingan awal.',
+    ],
+
+    'beatmapset_discussion' => [
+        'beatmap_missing' => 'Keterangan waktu telah ditentukan tetapi beatmap tidak ada.',
+        'beatmapset_no_hype' => 'Beatmap tidak dapat dihype.',
+        'hype_requires_null_beatmap' => 'Hype harus dilakukan di bagian Umum (semua tingkat kesulitan).',
+        'invalid_beatmap_id' => 'Tingkat kesulitan yang tidak valid ditentukan.',
+        'invalid_beatmapset_id' => 'Beatmap yang tidak valid ditentukan.',
+        'locked' => 'Diskusi dikunci',
+        'mapper_note_wrong_user' => 'Hanya pemilik beatmap yang dapat mengirim catatan mapper.',
+
+        'hype' => [
+            'guest' => 'Silakan masuk untuk hype.',
+            'hyped' => 'Anda telah melakukan hype untuk beatmap ini.',
+            'limit_exceeded' => 'Anda telah menggunakan semua hype Anda.',
+            'not_hypeable' => 'Beatmap ini tidak dapat dihype.',
+            'owner' => 'Tidak terdapat hype di beatmap Anda.',
+        ],
+
+        'timestamp' => [
+            'exceeds_beatmapset_length' => 'Keterangan waktu yang ditentukan melebihi panjang beatmap.',
+            'negative' => 'Keterangan waktu tidak bisa bernilai negatif.',
+        ],
+    ],
+
+    'forum' => [
+        'feature_vote' => [
+            'not_feature_topic' => 'Hanya dapat memilih permintaan fitur.',
+            'not_enough_feature_votes' => 'Suara tidak cukup.',
+        ],
+
+        'poll_vote' => [
+            'invalid' => 'Opsi tidak valid ditentukan.',
+        ],
+
+        'post' => [
+            'beatmapset_post_no_delete' => 'Menghapus posting metadata beatmap tidak diizinkan.',
+            'beatmapset_post_no_edit' => 'Menyunting posting metadata beatmap tidak diizinkan.',
+        ],
+
+        'topic_poll' => [
+            'duplicate_options' => 'Opsi ganda tidak diizinkan.',
+            'invalid_max_options' => 'Opsi per pengguna tidak boleh melebihi jumlah opsi yang tersedia.',
+            'minimum_one_selection' => 'Diperlukan setidaknya satu opsi per pengguna.',
+            'minimum_two_options' => 'Diperlukan setidaknya dua opsi',
+            'too_many_options' => 'Jumlah maksimum opsi melebihi yang diizinkan.',
+        ],
+
+        'topic_vote' => [
+            'required' => 'Pilih opsi saat memilih.',
+            'too_many' => 'Jumlah pilihan Anda lebih banyak dari yang diizinkan.',
+        ],
+    ],
+
+    'user' => [
+        'contains_username' => 'Nama pengguna tidak diperbolehkan untuk berada di dalam kata sandi.',
+        'email_already_used' => 'Alamat email sudah digunakan',
+        'invalid_country' => 'Negara tidak ada dalam basis data.',
+        'invalid_discord' => 'Nama pengguna Discord tidak valid.',
```

Comment:
since Discord uses "tags" instead of "usernames" this should work better in the context I think : 

+        'invalid_discord' => 'Tag pengguna Discord tidak valid.',

Reply:
iirc they use both. example for mine is 
>fajar13k#1494

And you must type it all with no spaces. You can see it'll display as "Username" in your Discord User Settings.